eras/babbage/test-suite/cddl-files/babbage.cddl

```diff
@@ -278,9 +278,10 @@ transaction_witness_set =
   { ? 0: [* vkeywitness ]
   , ? 1: [* native_script ]
   , ? 2: [* bootstrap_witness ]
-  , ? 3: [* plutus_script ]
+  , ? 3: [* plutus_v1_script ]
   , ? 4: [* plutus_data ]
   , ? 5: [* redeemer ]
+  , ? 6: [* plutus_v2_script ] ; New
   }
 
 plutus_script = bytes
```

Comment:
prolly update this as well. Also, I remember there was a discussion regarding using tagging for plutus script versions, that it could be easier to upgrade in future? was that idea dropped?

Reply:
found it, nvm, https://github.com/input-output-hk/cardano-ledger/issues/2601

Reply:
yep, good catch! Looks like the test caught it too :)